tests/test_mapbox_osrm.py

```diff
@@ -128,7 +128,7 @@ def test_full_isochrones(self):
 
         self.assertEqual(1, len(responses.calls))
         self.assertURLEqual(
-            'https://api.mapbox.com/isochrone/v1/mapbox/driving/8.34234,48.23424?costing=mapbox/driving&access_token=sample_key&'
+            'https://api.mapbox.com/isochrone/v1/mapbox/driving/8.34234,48.23424?driving&access_token=sample_key&'
```

Comment:
it passes the test but it's still not really correct:

```suggestion
            'https://api.mapbox.com/isochrone/v1/mapbox/driving/8.34234,48.23424?access_token=sample_key&'
```